tests/recovery/recovery_test.go

```diff
@@ -9,16 +9,27 @@ import (
 
 var _ = Describe("cOS Recovery upgrade tests", func() {
 	var s *sut.SUT
+
+	BeforeSuite(func() {
+		sut.SnapshotVagrant()
```

Comment:
Can't we make s.Reset smarter and check for e.g. if we are running on vagrant to use the snapshot feature instead of `cos-reset` directly ?

Reply:
yes, no problem there. I just wanted to see if this passes tests and cuts the time by half like in my local machine :o

Will add that!

Reply:
Done!  Now it checks at the suite start if vagrant is in a running state, and stores the value to use in the teardown